seqr/views/apis/dataset_api_tests.py

```diff
@@ -280,11 +281,11 @@ def test_add_alignment_sample(self, mock_local_file_exists, mock_google_bucket_f
 
         self.assertSetEqual(set(response_json.keys()), {'igvSamplesByGuid', 'individualsByGuid'})
         self.assertEqual(len(response_json['igvSamplesByGuid']), 1)
-        sample_guid = response_json['igvSamplesByGuid'].keys()[0]
+        sample_guid = list(response_json['igvSamplesByGuid'].keys())[0]
```

Comment:
while this works, I think the more pythonic way to do this is `sample_guid = next(guid for guid in response_json['igvSamplesByGuid'].keys())`

Reply:
Hmm..., I try using next() + iter(). Is it okay?

Reply:
Well theres no need to explicitly call `iter()`, the code I showed here implicitly calls it, but yes thats the idea

Reply:
`next(iter(response_json['igvSamplesByGuid']))` looks more concise. I'm going to use it.